.github/workflows/ci.yml

```diff
@@ -0,0 +1,30 @@
+name: CI
+
+on:
+  push:
+    branches: [master]
+  pull_request:
```

Comment:
forgive my ignorance, but does this configure it to run on all pull requests? or just a subset? the syntax seems a bit weird.